oss-fuzz-build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eu
```

Comment:
What is this file used for?

Reply:
This will be used by oss-fuzz to build the `fuzz` package when submit this repo to it.

Reply:
@adlerjohn See my comment https://github.com/celestiaorg/smt/pull/51#issuecomment-886845287 for more details.